device/transport/mqtt/devdoc/mqtt_requirements.md

```diff
@@ -80,37 +80,53 @@ The `sendEvent` method sends an event to an IoT hub on behalf of the device indi
 
 **SRS_NODE_DEVICE_MQTT_12_005: [** The `sendEvent` method shall call the publish method on `MqttTransport`. **]**
 
+**SRS_NODE_COMMON_MQTT_BASE_16_008: [** The `sendEvent` method shall use a topic formatted using the following convention: `devices/<deviceId>/messages/events/`. **]**
+
+**SRS_NODE_COMMON_MQTT_BASE_16_009: [** If the message has properties, the property keys and values shall be uri-encoded, then serialized and appended at the end of the topic with the following convention: `<key>=<value>&<key2>=<value2>&<key3>=<value3>(...)`. **]**
```

Comment:
><key> [](start = 207, length = 5)

Is there a ? before the first key as in HTTP URIs?

Reply:
no apparently it's after a '/' directly that is included in the topic name, specified in SRS_NODE_COMMON_MQTT_BASE_16_008

---
In reply to: [141150761](https://github.com/Azure/azure-iot-sdk-node/pull/90#discussion_r141150761) [](ancestors = 141150761)